src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -168,6 +177,51 @@ private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptP
             return scriptHostConfig;
         }
 
+        private static string MakeValidHostId(string id)
```

Comment:
I think all this logic to default the host ID to the local machine name should go in the CLI. I believe there is already logic in the CLI to default the host ID to machine name?

We've been running fine up to this point locally - the guidance is for people to simply put an ID in their host.config, as we do in the sample app.